projects/vdk-plugins/vdk-kerberos-auth/src/vdk/plugin/kerberos/api/kerberos_client.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from typing import Optional
+
+
+class KerberosClient:
+    """
+    Use this class to access prepared Kerberos Negotate header that can be used in HTTP API requests.
+    It's expected to be set as an "Authorization header"
+    Example usage
+
+    auth = KerberosClient("HTTP@vdk.fqdn.com")
+    api_call_headers["Authorization"] = auth.read_kerberos_auth_header()
```

Comment:
method is called `read_kerberos_auth_header` here, but `acquire_kerberos_auth_header` in the implementation.